File: After_Party_Task/AverageScore.java
import java.util.Scanner;
public class AverageScore{
public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    
    int totalScore = 0;
    int average = 0;
    
       
   
    for ( int counter =1; counter <= 3;counter++ ){ 
    System.out.println("Enter score");
          int score = input.nextInt();
          totalScore = totalScore + score;
          
          if (score>=90){
          
          System.out.println("You got A");
          }
          else if(score>=80){
           System.out.println("You got B");
          
          }
         else if(score>=70){
          System.out.println("You got C");
         
         }
         else if(score>=60 ){
          System.out.println("You got D");
         
         }
         
         else if(score<=60 ){
          System.out.println("You got F");
         
         }
          else{
          System.out.println("Bring your parent");
         
         }
         
         
    }
    average = totalScore/3;
    System.out.printf("Total score is : %d%n", totalScore);
     System.out.printf("Average score is : %d%n", average);
    
    
    
    }
    }

File: After_Party_Task/average_score.py
total_score = 0
average = 0
counter = 0
    
       
while counter < 3:
 
    print ("Enter score between 1 and 100")

    score =int(input("Enter score: " ))

    total_score = total_score + score
  
    if 90 <= score <= 100:
        print("You got A")
  
    elif 80 <= score <= 90:
        print("You got B")
  
    elif 70 <= score <= 80:
        print("You got C")
 
    elif 60 <= score <=70:
        print("You got D")
 
    elif 0 <= score <= 60:
        print("You got F") 
    else:
        print("Bring your parent")
    counter +=1    
 
 

average = total_score / 3;
print("Total score is : ", total_score)
print("Average score is : ", average)





